### ISSUE TITLE
Training curve

### ISSUE THREAD

**Issue by cxjyxxme** [MingtaoFu/gliding_vertex#44]
Hello, we tried to train this model in cuda11, due to GPU limitations. The model can be successfully trained, but the mAP of the test set and the training set are relatively low after training. The mAP is only about 0.3. We suspect it is caused by the cuda version. Do you know what might cause this result? Or can you please provide the loss curve on tensorboard in the default configuration as a reference? Thank you very much!

Here is our training script:
`python3 --config-file configs/glide/dota.yaml --use-tensorboard`

**Comment by cxjyxxme** [comment]
Seem to caused by a pytorch 1.8.0+cu111 bug.
https://github.com/pytorch/pytorch/issues/54245
Solved by use pytorch 1.7.1+cu110, and replace torch.uint8 by torch.bool.
Get mAP with about 0.75 in DOTA1.0 testing set.
